packages/components/src/scrollable/component.js

```diff
@@ -1,15 +1,15 @@
 /**
  * Internal dependencies
  */
-import { createComponent } from '../utils';
+import { createComponent } from '../ui/utils';
 import { useScrollable } from './hook';
 
 /**
  * `Scrollable` is a layout component that content in a scrollable container.
  *
  * @example
  * ```jsx
- * import { Scrollable } from `@wordpress/components/ui`;
+ * import { __experimentalScrollable as Scrollable } from `@wordpress/components/ui`;
```

Comment:
docs/manifest.json

Reply:
Hey @nenel1221 , I'm not sure I'm getting your comment — could you elaborate, please?